test/NuGet.Core.Tests/NuGet.Credentials.Test/PluginManagerMock.cs

```diff
@@ -66,9 +66,21 @@ internal TestExpectation(
 
     internal sealed class PluginManagerMock : IDisposable
     {
+        public static bool IsDesktop
```

Comment:
I think all of these `IsDesktop` properties would be better off named `RequiresDotNetHost`, since that's how it's used.  `IsDesktop` is a little more difficult to understand than `RequiresDotNetHost`.  It would simplify call sites too.  Instead of `requiresDotnetHost: !IsDesktop`, you could simply pass `RequiresDotNetHost`.

Your call though.